common/src/service-connectors/core/schemas/MemberRestriction.schema.ts

```diff
@@ -25,4 +30,4 @@ export const MemberRestrictionSchema = z.object({
     memberId: z.number(),
 });
 
-export type MemberRestrictionSchemaType = z.infer<typeof MemberRestrictionSchema>;
```

Comment:
This name conflicts at point of use elsewhere in core (there is already a type `MemberRestriction` from the database model). hence why I named it wonky. Could be a better name, but it really shouldn't be `MemberRestriction`

Reply:
Was this discussed outside of Github?

Reply:
Yeah. As far as other microservices are aware, core models don’t exist and these are “that thing”. If they’re needed in core, it can follow the same pattern as scrims and do ISomeThing. We’ve got conflicts in core itself anyway between database and other modules.